### ISSUE TITLE
Distance Limit from Kerbol

### ISSUE THREAD

**Issue by horstdieta** [mockingbirdnest/Principia#1737]
As of now, Principia only seems to work if you are pretty much inside the Orbit of Eelo. If you go further, Principia stops showing any predictions, and you can´t create new flight plans. This prevents me from using the mod together with the Outer Plantets mod or even Extrasolar. Is there an easy way to increase this pesky limit or to remove it completely?

**Comment by pleroy** [comment]
There is no "pesky limit" or anything like that.  However what you are seeing is an interesting effect.  We never tried to go beyond Eeloo so we never saw it, but now that we tried we've encountered the problem.  Here is what we think is happening:

When the vessel goes very far from Kerbol, you tend to zoom out a lot to get the big picture.  This means that the prediction looks smaller and smaller on the screen, and viewing a sizeable prediction requires to compute more and more steps into the future.  At the same time the vessel moves slower and slower and that also increases the number of steps that need to be computed to display a sizeable prediction.  All this requires to integrate the entire solar system as well as the vessel over extremely long periods of time and at some point this becomes so costly that we cannot keep up and the prediction looks smaller and smaller (although it's longer and longer in absolute terms, i.e. in number of kilometers).

I don't see that this is something that we can ever hope to fix.  We can and do try to optimize things to make the game more pleasant to use, but interstellar travel is just beyond what can be done on a single computer.  Space is too big.
